.github/workflows/ci.yml

```diff
@@ -84,12 +84,24 @@ jobs:
       - name: Cache
         uses: Swatinem/rust-cache@v2
 
+      - name: Install, run latest `substrate-contracts-node` binary and run tests
+        if: runner.os == 'Linux'
+        run: |
+          curl -L -o substrate-contracts-node.zip 'https://gitlab.parity.io/parity/mirrors/substrate-contracts-node/-/jobs/artifacts/main/download?job=build-linux' && \
+          unzip substrate-contracts-node.zip && \
+          chmod +x artifacts/substrate-contracts-node-linux/substrate-contracts-node &&
+          ./artifacts/substrate-contracts-node-linux/substrate-contracts-node -lruntime::contracts=debug  2>&1 | tee /tmp/contracts-node.log &&
```

Comment:
Note that the integration tests themselves spawn (or attempt to) `substrate-contracts-node` on the path. So they should still work if you remove this line?

Reply:
Yep it seems to work